public/css/style.css

```diff
@@ -7,6 +7,96 @@ body {
   justify-content: center;
 }
 
+#loader {
+
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  z-index: 1;
+  width: 150px;
+  height: 150px;
+  margin: -75px 0 0 -75px;
+  border: 16px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 16px solid #3498db;
+  width: 120px;
+  height: 120px;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+  display: none;
+}
+.likes {
```

Comment:
The likes are good idea!
can you just fix the alignments of the likes? 

add maybe a div for the likes and the h4 number of likes so they will be aligned `align-items: center;`

Reply:
yes there was styling issues!, now fixed ! :tada: :tada: